plugins/modules/ec2_eip.py

```diff
@@ -18,28 +18,28 @@
     description:
       - The id of the device for the EIP.
       - Can be an EC2 Instance id or Elastic Network Interface (ENI) id.
-      - When specifying an ENI id, O(in_vpc) must be V(true).
```

Comment:
These changes to the documentation block should be reverted, the semantic markup was correct.

Reply:
I forgot to rebase my PR